docs/TestCoverage.md

```diff
@@ -5609,6 +5609,9 @@ expect(node, inputs=[x, y], outputs=[z],
 <br/>
 
 ## &#x1F494;No Cover Common Operators
+### ConvInteger (call for test cases)
```

Comment:
Functionality of these ops is the same as their original ones (ones for full-precision). Only differences are data types, which are defined by type constraints and type inference function. besides adding same node test cases, any other test cases suggested to add please?

Reply:
am adding test data for convinteger. cases for the other two will be added separately. For the type checking, it will need some test API change per discussion with @houseroad  offline.